source/fuzz/transformation_replace_constant_with_uniform.cpp

```diff
@@ -90,6 +90,40 @@ TransformationReplaceConstantWithUniform::MakeLoadInstruction(
                                       operands_for_load);
 }
 
+opt::Instruction*
+TransformationReplaceConstantWithUniform::GetInsertBeforeInstruction(
+    opt::IRContext* ir_context) const {
+  auto* result =
+      FindInstructionContainingUse(message_.id_use_descriptor(), ir_context);
+  if (!result) {
+    return nullptr;
+  }
+
+  // OpPhi must be the first instruction in the module. Thus, we can't insert
+  // anything above it.
+  if (result->opcode() == SpvOpPhi) {
+    // Given the predecessor block that is dominated by the replaced id (the id
+    // of such a block is one of the OpPhi's operands), get the last instruction
+    // from that block that we can use to insert OpAccessChain and OpLoad above.
+    return fuzzerutil::GetLastInsertBeforeInstruction(
+        ir_context,
+        result->GetSingleWordInOperand(
+            message_.id_use_descriptor().in_operand_index() + 1),
+        SpvOpLoad);
+  }
+
+  // The only operand that has a scalar value is a condition id.
```

Comment:
The only operand that has a boolean type?

Reply:
Clarified the comment in bcb5394.